cmake/winml_unittests.cmake

```diff
@@ -142,7 +143,7 @@ add_winml_test(
   SOURCES ${winml_test_api_src}
   LIBS winml_test_common delayimp.lib
 )
-target_link_options(winml_test_api PRIVATE /DELAYLOAD:dxgi.dll /DELAYLOAD:d3d12.dll /DELAYLOAD:api-ms-win-core-file-l1-2-2.dll /DELAYLOAD:api-ms-win-core-synch-l1-2-1.dll)
+target_link_options(winml_test_api PRIVATE /DELAYLOAD:d3d11.dll /DELAYLOAD:dxgi.dll /DELAYLOAD:d3d12.dll /DELAYLOAD:api-ms-win-core-file-l1-2-2.dll /DELAYLOAD:api-ms-win-core-synch-l1-2-1.dll)
```

Comment:
Is this to handle video frames backed by d3d11 resource?

Reply:
Actually no, the Raw Api tests added a test for creating a d3d11device which added an import to d3d11 on the apitests dll that didn't exist previously.